pkg/liqonet/iptables/iptables.go

```diff
@@ -683,13 +726,26 @@ func checkTep(tep *netv1alpha1.TunnelEndpoint) error {
 			Reason:    errors.ValidCIDR,
 		}
 	}
+	if err := utils.IsValidCIDR(tep.Status.LocalExternalCIDR); err != nil {
```

Comment:
It would be better to move the `checkTep` function in the `utils` package. We could use it in all the network operators to validate the `tep` instances.